node_modules/col-assets/lib/rest.js

```diff
@@ -21,7 +21,7 @@ var AssetsAPI = require('./api');
  * Get an asset
  */
 Collabosphere.apiRouter.get('/assets/:assetId', function(req, res) {
-  AssetsAPI.getAsset(req.ctx, req.params.assetId, function(err, asset) {
+  AssetsAPI.getAsset(req.ctx, req.params.assetId, req.query.increment_views, function(err, asset) {
```

Comment:
I was thinking that the UI might want to request the asset without increasing the views (e.g. reloading the asset after making a comment). However, having done the REST APIs for comments, this is probably no longer necessary and we can just increase the views for every REST call.